src/content/docs/tutorial-customer-experience/intro-to-customer-experience.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: "Improve your customers' digital experience with New Relic" 
+metaDescription: "New Relic observability maturity series: this guide gives you a quality foundation for optimizing your web application's performance to improve your customer experience."
+redirects:
+  - /docs/new-relic-solutions/observability-maturity/qf-implementation-guide
+  - /docs/new-relic-solutions/observability-maturity/customer-experience/quality-foundation-implementation-guide
+  - /docs/new-relic-solutions/observability-maturity/customer-experience/customer-experience-quality-foundation-guide
+---
+
+import cxWhatYouCanMeasureNr from 'images/cx_diagram_what-you-can-measure-nr.webp'
+
+Digital customer experience refers to the quality of how your users interact with your service across all your digital touch points (web, mobile, and IoT). Because it has so many moving parts within the concept, it can be difficult to digest when taken as whole. You need to manage the availability of your service, its performance to ensure that it's usable, the quality of the content that gives users the information they need, and the relevance of your products and content so that users remain engaged.
+
+New Relic can help you make managing this much more approachable by measuring and improving performance in a way that better aligns to the end user experience. This guide covers the procedures that help you understand your digital customer experience in a meaningful way:
+
+* Look at customer experience in relation to:
```

Comment:
I don't love the colon phrase leading into a bullet with a colon phrase. 

It's not clear to me how "...understand your digital customer experience in a meaningful way:" leads into these bullets in a meaningful way. 

Is there some more direct context you can add to lead into these bullets?

Reply:
Agreed, I'll reformat this and try to solidify the connection. Thank you!